pallet.el

```diff
@@ -237,6 +238,32 @@ use `pallet--package-archives-copy' if USE-COPY is true."
       (message "Pallet: unpacking %s" package-name)
       (pallet--unpack-one package-name))))
 
+;; Command to remove unrequested packages
+(defun pallet-auto-remove-packages ()
```

Comment:
On second thoughts: for the sake of argument let's say we keep this (or Cask gives us a proxy for it), and it just uninstalls surplus packages. In that case I think `pallet-prune` is a good name for the function (in harmony with e.g. `npm prune`.)